app.js

```diff
@@ -18,7 +18,7 @@ registerListeners(app);
 /** Start the Bolt App */
 (async () => {
   try {
-    await app.start();
+    await app.start(process.env.PORT || 3000);
```

Comment:
```suggestion
    await app.start();
```

🪬 question: Do we need to also include this for Socket Mode when not using OAuth?